tv_apps/smart-home/js/home.js

```diff
@@ -662,6 +662,10 @@
         this._cardPicker.saveToNewFolder(this.cardScrollable.currentIndex);
       } else if (this._cardPicker.mode === 'update') {
         this._cardPicker.updateFolder();
+        var folder = this._cardPicker.folder;
+        if (folder.isEmpty()) {
+          this.cardManager.removeCard(folder);
```

Comment:
Please move this manipulation into `cardPicker`.

Supposedly to be in `_saveToFolderHelper`.